smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -395,6 +418,14 @@ public boolean isLevel() {
     return this.areAllLevel(RecentPitches) && this.areAllLevel(RecentRolls);
   }
 
+  /**
+   * A method to determine if the power reduction factor is maximum
+   * @return true if is maximum, false otherwise
+   */
+  public boolean isMaxPowerReductionFactor() {
+    return speedReductionFactor == maximumSpeedReductionFactor;
```

Comment:
Curious to know if we have a caller here for this method yet?

Reply:
Yes. In DriveRampUpSpeedCommand.java.